### ISSUE TITLE
multiline parser doesn't concatenate lines into a single event in fluent-bit 1.9.2 (k8s env)

### ISSUE THREAD

**Issue by nickytd** [fluent/fluent-bit#5358]
## Bug Report

**Describe the bug**
In k8s env, when multiline parser in fluent-bit is used to aggregate stacktraces it generates multiple events instead of a single such.

- Example log message:
```
[0] java: [1650887767.901349306, {"stream"=>"stderr", "_p"=>"F", "log"=>"java.lang.Exception: Stack trace"}]
[1] java: [1650887767.901349306, {"log"=>"	at java.base/java.lang.Thread.dumpStack(Thread.java:1377)"}]
[2] java: [1650887767.901349306, {"log"=>"	at MyException.dump(MyException.java:6)"}]
[3] java: [1650887767.901349306, {"log"=>"	at MyException.main(MyException.java:12)"}]
```

**Expected behavior**
It shall generate single evemt. It was working in fluent-bit 1.8.13
```
[0] java: [1650888195.166662100, {"log"=>"	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:516)"}]
```

**Your Environment**
* Version used: 1.9.2
* Environment name and version:
  * kubernetes 1.23.4 (kind) 
* Filters and plugins:
  * tail 
  * kubernetes
  * multiline
  * stdout

**Comment by trallnag** [comment]
Does it work sometimes or never? There are currently a few other issues open regarding multiline parsing not working properly.

**Comment by nickytd** [comment]
Fixed in 1.9.3
